### PR TITLE
Fix some nits around the playbar

### PR DESCRIPTION
**Pull request by bcasselsND**
Use 0.001 increments for time.
Reflect changes in time window values, right away.
Reflect changes in elevation window, immediately when the mode is changed.
Split button text into multiple lines, so it fits better, when the panel is open.
Clean up code, a little.
Fix fonts, for some things.
Part of #134

### CODE REVIEW

File: common/playbar.js [hunk 1]
```diff
@@ -350,11 +338,12 @@ function createPlaybarListeners(playbarhtml) {
 			label.innerHTML = "NONE";
 		}
 	});
+  addPlaybarListeners();
 }
 
 
 // adds event listeners to animate the viewer and playbar's slider and play/pause button
-export function addPlaybarListeners() {
+function addPlaybarListeners() {
```
Comment (NickRizzo-droid) [inline common/playbar.js:346]: good call on not exporting this function

File: common/playbar.js [hunk 2]
```diff
@@ -100,33 +120,7 @@ export function createPlaybar () {
...
-
-    playbarhtml.find("#elevation_min").on('input', function() {
-      const elevationMin = playbarhtml.find("#elevation_min");
-      window.elevationWindow[0] = Math.abs(Number(elevationMin.val()));
-    });
-
-    playbarhtml.find("#myRange").on('input', function() {
-      updateSlider();
-    });
+    playbarhtml.find("#myRange").on('input', updateSlider);
```
Comment (NickRizzo-droid) [inline common/playbar.js:123]: Was this code not needed / causing bugs? What was the motivation for pulling it out?

Comment (bcasselsND) [reply common/playbar.js:123]: Unneeded. For elevation_min and _max, the things they are setting, window.elevationWindow[0] and [1], are not referenced anywhere. The (new) ways those effects happen are elsewhere.

Those 4 handlers, for elevation_min and _max, playbar_tmin and _tmax, were rewritten, to not use JQuery. And they were changed to allow positive and negative values, and to go up to the other value (min can go up to current max, and vice versa). But they are still present.

Comment (NickRizzo-droid) [reply common/playbar.js:123]: Ahh gotcha that makes sense!

**Comment by NickRizzo-droid** [comment]
Looks good to me

**Comment by bcasselsND** [comment]
Oops. Screwed up the merge. Try again.

**Pull request merged**